crates/spirv-std/src/lib.rs

```diff
@@ -1,7 +1,10 @@
 #![no_std]
-#![feature(register_attr, repr_simd, core_intrinsics, lang_items)]
+#![cfg_attr(
+    target_arch = "spirv",
+    feature(register_attr, repr_simd, core_intrinsics, lang_items)
+)]
 #![cfg_attr(target_arch = "spirv", feature(asm))]
-#![register_attr(spirv)]
+#![cfg_attr(target_arch = "spirv", register_attr(spirv))]
```

Comment:
I think this can be a single `cfg_attr` now.

Reply:
Missed this comment yesterday :)